### ISSUE TITLE
Factors for Fine-Tuning

### ISSUE THREAD

**Issue by Bartleby2718** [Bartleby2718/DieOrDare#155]
- NN
  - layers
  - optimizer
  - loss function
  - parameters in `model.fit()`
- epsilon
- discount rate
